### ISSUE TITLE
Accessibility of getting started editor

### ISSUE THREAD

**Issue by JacksonKearl** [microsoft/vscode#115896]
Getting started editor is now nicely keyboard navigable, but there are more improvements to be made. 

Running list:
- [x] Some way to announce images
- [x] Ensure collapsible button model works with screen readers
- [x] Ensure the overall navigation story makes sense (if the welcome editor is getting started, is it possible to figure out where you are and what to do via only the screen reader?)

**Comment by JacksonKearl** [comment]
cc @isidorn for suggestions!

**Comment by isidorn** [comment]
@JacksonKearl thanks for creating this issue.
Can you please share what setting I need to set in order to get the getting started editor. I tried to get to it last week but failed.

Then I can give it a spin and provide more feedback.
Once we have it in a nice spot, we can also ask for community feedback.

**Comment by JacksonKearl** [comment]
`workbench.experimental.gettingStarted`, then it will be in help menu and command palette.

**Comment by isidorn** [comment]
@JacksonKearl great, I successfuly enabled it. I think it looks cool, and it would be great to maybe share it in the UX channel so Zurich people can try it out and comment. Regarding accessibility, here are the items that I think we should tackle (let me know if you would prefer that I create separate issues for this):
- [ ]  Arrow navigation should just work
- [ ] `esc` should have the same effect as `skip`
- [ ] set `aria-expanded` on collapsable buttons. `aria-expanded` should be `true` when they are expanded, `false` when collapsed
- [ ] set appropriate [aria `role`](https://w3c.github.io/aria/#namefromauthor) on each element
- [ ] "Learn the fundamentals" part can not be focused thus it is hidden from screen reader users. Maybe set an aria-label somewhere to also capture that content for screen reader users or make it focusable
- [ ] Same for "Quick Setup" top part

I also have some general ux suggestions, but maybe you present this in the UX meeting (unless you have already).
Once you tackle this I can give it another pass. Thanks for working on this.

fyi @misolori

**Comment by misolori** [comment]
@isidorn we've actually have brought this up a couple of times during our ux sync so feel free to file any issues or start a discussion in our ux channel if there's anything specific.

**Comment by isidorn** [comment]
@misolori oh I thought that we brought up early slides, but not the final looks. I might have missed one of the meetings. No worries, I will file issues if I have suggestions.

**Comment by JacksonKearl** [comment]
Thanks for all the feedback! Implemented most of them but I was unable to repro not being able to focus the main content buttons, in my testing they focused fine and voceover read them as expected. Closing this out, please feel free to open additional a11y followups as new issues.

**Comment by isidorn** [comment]
Lovely thanks! I'll give it another spin, but maybe the best would be once you create a test plan item for this that you assign me for testing. So i can test it in general and checkout a11y state
